docker/pravega/Dockerfile

```diff
@@ -16,7 +16,11 @@ RUN apt-get update && apt-get install -y -q \
     jq \
     curl \
     net-tools \
-    iproute2  \
+    iproute2
+
+# Addidng ca-certificates to the base image
```

Comment:
Nit: 
* `Addidng` -> `Adding`. 
* A more useful comment'd be one that explains why you are adding it. Such as "Adding Java system truststore 'cacerts', as it is missing from the Java distribution installed by the base image". 
* Also, explain why "mkdir -p /usr/share/man/man1" is needed.